<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5Table</class>
 <widget class="QWidget" name="CE5Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="Primary" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leConfig">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_special_config</string>
     </property>
     <property name="Type" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="C5LineEditWithSelector" name="lineEdit_3">
     <property name="Field" stdset="0">
      <string>f_name</string>
     </property>
     <property name="First" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="C5LineEditWithSelector" name="leConfigName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>f_hallname</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="leHallCode">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_hall</string>
     </property>
     <property name="Type" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Hall</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Special config</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="C5LineEditWithSelector" name="leHallName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>f_hallname</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Hourly fee id</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="C5LineEditWithSelector" name="leConfig_2">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_hourly</string>
     </property>
     <property name="Type" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leHallCode</tabstop>
  <tabstop>leConfig</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>leHallName</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>leConfigName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
